FastAndFurious/FFRaceTrackViewController.m

```diff
@@ -89,6 +89,7 @@ - (NSString *)idForWinner {
         float finishLineXValue = self.view.frame.size.width - view.frame.size.width;
         if (x >= finishLineXValue && x >= furthestX) {
             idOfFurthest = identifier;
+            furthestX = x;
```

Comment:
Yeah, but furthestX is never updated from` 0`? It's only set to `0` in the current code.